astropy/io/misc/tests/test_hdf5.py

```diff
@@ -23,11 +23,11 @@
 
 ALL_DTYPES = [np.uint8, np.uint16, np.uint32, np.uint64, np.int8,
               np.int16, np.int32, np.int64, np.float32, np.float64,
-              np.bool, '|S3']
+              bool, '|S3']
```

Comment:
It doesn't really matter, but I think this more logically is `np.bool_` (or is alias, `np.bool8`)